helix-view/src/editor.rs

```diff
@@ -311,7 +313,13 @@ impl Editor {
     pub fn new_file(&mut self, action: Action) -> DocumentId {
         let id = DocumentId(self.next_document_id);
         self.next_document_id += 1;
-        let mut doc = Document::default();
+        let mut doc = if stdin().is_tty() {
```

Comment:
I don't think we want it to be in this function, I think it would make more sense for it to be in `helix-term/src/application.rs::Application::new()`

It did make me think of potentially opening new files that were piped from an application via `:n "<shell stuff>"` as a shortcut to creating a new file and directly piping it like that. @archseer